gallery/src/components/NxCombobox/NxComboboxDisabledExample.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useCallback, useState } from 'react';
+import { filter, map, prepend, range } from 'ramda';
+import { NxCombobox, DataItem }
+  from '@sonatype/react-shared-components';
+
+const items = prepend(
+    { id: 0, displayName: 'Loooooooooooooooooooooooooong Name' },
+    map(i => ({ id: i, displayName: `Item ${i}` }), range(1, 101))
+);
+
+function search(query: string):DataItem<number, string>[] {
+  const lowercaseQuery = query.toLowerCase(),
+      matchingItems = filter(i => i.displayName.toLowerCase().includes(lowercaseQuery), items);
```

Comment:
Should check prefixes only

Reply:
Related to the comments here https://github.com/sonatype/sonatype-react-shared-components/pull/751#discussion_r970039999 and it is updated.